snmp/tests/conftest.py

```diff
@@ -17,6 +17,13 @@
     "https://ddintegrations.blob.core.windows.net/snmp/3850.snmprec",
 ]
 
+E2E_METADATA = {
+    'start_commands': [
+        # Ensure the Agent has access to profile definition files.
+        'cp -r /home/snmp/datadog_checks/snmp/data/profiles /etc/datadog-agent/conf.d/snmp.d/',
```

Comment:
If possible, would be better to copy at ddev level (a step that will mimic what's done by the agent build phase)